ubcpi/ubcpi.py

```diff
@@ -627,7 +627,8 @@ def get_data(self,data,suffix=''):
         """
         Retrieve persisted date from backend for current user
         """
-        return self.get_persisted_data()
+        return self.get_persisted_data(get_other_answers(
```

Comment:
where do you ensure the selected answers from others are the same ones that you persisted above (line 535)?

Reply:
This is the existing way of retrieving other answers to display, but looks to be a bug, as a refresh can show a different set of answers (also tested on a version of PI prior to the commit for this pull request)--will have to implement a fix.

Reply:
@xcompass I have implemented a fix for this, and updated the pull request.